packages/atomic/src/components/facets/facet-date-input/facet-date-input.tsx

```diff
@@ -94,6 +94,8 @@ export class FacetDateInput {
           class={inputClasses}
           aria-label={startAria}
           required
+          // API/Index minimum supported date
+          min={this.formattedDateValue('1401-01-01')}
```

Comment:
If this wouldn't be there, the app still doesn't crash, we just see an error in the console as the failed action threw it. I just thought it would be a better dev experience to add a limit directly here, has a bonus